services/cnae-code/cnaeCodeList.json

```diff
@@ -0,0 +1,5433 @@
+[
```

Comment:
Dúvida mestre, conheço pouco desse domínio mas a natureza desses dados se altera muito? To perguntando pra saber se vale a pena a gnt caçar uma fonte confiável e atualizada dos dados 👀

Reply:
Acredito que não, como é uma informação sobre a formação societária de uma empresa acho meio difícil o governo ficar criando novos tipos